File: python/7 kyu/405.py
# https://www.codewars.com/kata/51c38e14ea1c97ffaf000003/train/python

# Passed

def populate_dict(keys, default):
    result = dict.fromkeys(keys, default)
    return result

output = populate_dict(["draft", "completed"], 0)
print(output)

File: nodejs/7 kyu/405.js
// https://www.codewars.com/kata/5829994cd04efd4373000468/train/javascript

// Passed

function nameFile(fmt, nbr, start) {
  let result = [];
  let isWhole = (numToCheck) => Math.ceil(numToCheck) === numToCheck;
  if (nbr > 0 && isWhole(nbr) && isWhole(start)) {
    for (let indexNum = start; indexNum < start + nbr; indexNum++) {
      let fileName = fmt.replaceAll("<index_no>", String(indexNum));
      result.push(fileName);
    }
  }

  return result;
}

const output = nameFile("IMG <index_no>", 4, 1);
console.log(output);